.ci/jenkins/Jenkinsfile.release

```diff
@@ -252,11 +253,11 @@ pipeline {
 				        env.STAGING_STATUS = ""
 				        counter=0
 				        while( "${env.STAGING_STATUS}" != "closed" ){
-    				        env.STAGING_STATUS = sh(script: """ curl -u $CREDS -H 'Accept: application/json' -H 'Content-Type: application/json' https://repository.jboss.org/nexus/service/local/staging/profile_repositories/15c58a1abc895b | grep -oP '$repoID\",\"type\":\"\\K[^\"]+(?=\")' """, returnStdout: true).trim()
-    				        echo "staging-repository is still ${STAGING_STATUS}"
-    				        sleep(60*15)
-    				        counter=counter+1
-    				        echo "counter: $counter"
+                            env.STAGING_STATUS = sh(script: """ curl -u $CREDS -H 'Accept: application/json' -H 'Content-Type: application/json' https://repository.jboss.org/nexus/service/local/staging/profile_repositories/15c58a1abc895b | grep -oP '$repoID\",\"type\":\"\\K[^\"]+(?=\")' """, returnStdout: true).trim()
+                            echo "staging-repository is still ${STAGING_STATUS}"
+                            sleep(60*15)
+                            counter=counter+1
+                            echo "counter: $counter"
 				        }
```

Comment:
why this reformatting? @mbiarnes

Reply:
@mareknovotny  because zip of the org.kie bits are to big and has to be splitted. This wass there before ! I made a mistake and put in a previous PR https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2297/files#diff-9d549ac932d95243b78185fd4d6662b971306a13c87c95ea4e00acecb33c666cR232 `kie` here
The split itself works (at least in the previous release and in the last test)